### PR TITLE
Bug 1261061: Stop using the "dbo" wrapper in db.py

### PR DESCRIPTION
**Pull request by andela-oadeniran**
## What does this PR DO?
 It eliminates the use of dbo wrapper in the db.py file.
## Description of tasks to be Completed
Use db passed to each class, instead of dbo which is an instance of the DBWrapper.

### CODE REVIEW

File: auslib/db.py [hunk 1]
```diff
@@ -2572,7 +2573,7 @@ def create(self, version=None):
 
     def upgrade(self, version=None):
         # This method was taken from Buildbot:
-        # https://github.com/buildbot/buildbot/blob/87108ec4088dc7fd5394ac3c1d0bd3b465300d92/master/buildbot/db/model.py#L455
+        # https://github.com/buildbot/buildbot/blob/87108ec4088dc7fd5394ac3c1d0bd3b465300d92/master/builselft/db/model.py#L455
```
Comment (NinadBhat) [inline auslib/db.py]: Any reason you changed this url?

Comment (andela-oadeniran) [inline auslib/db.py]: It was a typo I would correct that immediately

File: auslib/db.py [hunk 2]
```diff
@@ -20,7 +20,7 @@
 import dictdiffer
 import dictdiffer.merge
 
-from auslib.global_state import cache, dbo
```
Comment (njirap) [inline auslib/db.py]: did you guys delete `dbo` from `auslib.global_state` or is it being used elsewhere apart from here?

Comment (andela-oadeniran) [inline auslib/db.py]: We thought of that but it is used in other files.

Comment (mozbhearsum) [inline auslib/db.py]: Yeah, we can't get rid of it entirely - it's required by the web layer. Good thought though!

**Pull request merged**